phpBB/docs/README.html

```diff
@@ -91,24 +91,25 @@
 		<div class="content">
 	<p>Installation, update and conversion instructions can be found in the <a href="INSTALL.html">INSTALL</a> document in this directory. If you are intending on converting from a phpBB 2.0.x or 3.0.x installation we highly recommend that you backup any existing data before proceeding!</p>
 
-	<p>Users of phpBB 3.0 and 3.1 Beta versions cannot directly update.</p>
+	<p>Users of phpBB 3.0, 3.1 and 3.2 Beta versions cannot directly update.</p>
 
 	<p>Please note that we don't support the following installation types:</p>
 	<ul>
-		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.0 RC1 and higher</li>
+		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.1 RC1 and higher</li>
```

Comment:
In general we should look through the docs and check if we can improve on the portability between versions. For example the build script does now always use the version names (i.e. ascraeus, Rhea) while we should really try to eliminate that

Am 06.06.2016 7:53 vorm. schrieb Tristan Darricau <notifications@github.com>:In phpBB/docs/README.html:
>  
>  	<p>Please note that we don't support the following installation types:</p>
>  	<ul>
> -		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.0 RC1 and higher</li>
> +		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.1 RC1 and higher</li>

If it's not tight to 3.0 (dont have the whole text right now) we could be more general and say "update from beta to RC"

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.